tic_tac_toe.py

```diff
@@ -1,14 +1,16 @@
 from ttt.console_ui.consoleio import ConsoleIO
 from ttt.console_ui.console import Console
 from ttt.console_ui.menu import Menu
+from ttt.game.game_runner import GameRunner
 
 consoleio = ConsoleIO()
 console = Console(consoleio)
-interface = Menu(console)
+runner = GameRunner(console)
+menu = Menu(console, runner)
```

Comment:
👍 

This looks really good and no unnecessary dependencies being injected. Nice one.

Have a think about whether these need to be global or if they can be scoped to `main`.